temporaltest/server.go

```diff
@@ -181,5 +183,25 @@ func NewServer(opts ...TestServerOption) *TestServer {
 		ts.fatal(err)
 	}
 
+	// This sleep helps avoid the following panic:
```

Comment:
The comment may be a little excessive.. you could just say it avoids a panic in ringpop